diff -uNdr mxser.old/driver/kernel5.x/Makefile mxser/driver/kernel5.x/Makefile
--- mxser.old/driver/kernel5.x/Makefile	2019-09-04 05:27:57.000000000 +0300
+++ mxser/driver/kernel5.x/Makefile	2020-10-21 15:59:46.452163116 +0300
@@ -1,4 +1,4 @@
-MXVER:=`awk '{if($$2=="MX_SER_VERSION"){print $$3}}' ../../mx_ver.h | sed s/\"//g`
+MXVER:=`awk '{if($$2=="MX_SER_VERSION"){print $$3}}' mx_ver.h | sed s/\"//g`
 TARGET_DRIVER1:=mxser
 TARGET_DRIVER2:=mxupcie
 CPLD_PATH:=../CPLD/Source
diff -uNdr mxser.old/driver/kernel5.x/mxconf mxser/driver/kernel5.x/mxconf
--- mxser.old/driver/kernel5.x/mxconf	2019-09-04 05:27:57.000000000 +0300
+++ mxser/driver/kernel5.x/mxconf	2020-10-21 15:59:46.453163130 +0300
@@ -17,12 +17,12 @@
 ###############################################################################
 
 #Parameters
-MX_MSB_VERSION=`awk '{if($2=="MX_SER_VERSION"){print $3}}' ../../mx_ver.h | sed s/\"//g`
+MX_MSB_VERSION=`awk '{if($2=="MX_SER_VERSION"){print $3}}' mx_ver.h | sed s/\"//g`
 MX_RELEASE=5.0.0
 MX_VERSION=5
 MX_PATCH_LEVEL=0
 MX_SUBLEVEL=0
-MX_RELEASE_DATE=`awk '{if($2=="MX_SER_DATE"){print $3}}' ../../mx_ver.h | sed s/\"//g`
+MX_RELEASE_DATE=`awk '{if($2=="MX_SER_DATE"){print $3}}' mx_ver.h | sed s/\"//g`
 MX_VERSION_CODE=$(( ($MX_VERSION<<16)+($MX_PATCH_LEVEL<<8)+$MX_SUBLEVEL ))
 
 K_MODULE_VERSION=$(uname -r)
diff -uNdr mxser.old/driver/kernel5.x/mxpcie.c mxser/driver/kernel5.x/mxpcie.c
--- mxser.old/driver/kernel5.x/mxpcie.c	2019-09-04 05:27:57.000000000 +0300
+++ mxser/driver/kernel5.x/mxpcie.c	2020-10-21 16:27:00.103762869 +0300
@@ -27,7 +27,7 @@
 
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
-#include "../../mx_ver.h"
+#include "mx_ver.h"
 
 #include <linux/version.h>
 #define	VERSION_CODE(ver,rel,seq)	((ver << 16) | (rel << 8) | seq)
diff -uNdr mxser.old/driver/kernel5.x/mxser.c mxser/driver/kernel5.x/mxser.c
--- mxser.old/driver/kernel5.x/mxser.c	2019-09-04 05:27:57.000000000 +0300
+++ mxser/driver/kernel5.x/mxser.c	2020-10-21 15:59:46.453163130 +0300
@@ -16,7 +16,7 @@
 
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
-#include "../../mx_ver.h"
+#include "mx_ver.h"
 
 #include <linux/version.h>
 #define	VERSION_CODE(ver,rel,seq)	((ver << 16) | (rel << 8) | seq)
